crates/sui-framework/packages/dwallet-system/sources/dwallet.move

```diff
@@ -59,7 +59,7 @@ module dwallet_system::dwallet {
 
     /// `MessageApproval` represents a message that was approved.
     /// Bound to a `DWalletCap`.
-    struct MessageApproval has store {
+    struct MessageApproval has store, drop {
```

Comment:
why did you add this? @omersadika

Reply:
last line of the `approve_message` is calling:
`dwallet::approve_messages(&eth_dwallet_cap.dwallet_cap, vector[message])`.
When there is no drop ability I get a runtime error whenever i approve a message, even though my function also returns `vec<MessageApproval>`